src/pages/Pokedex/Pokedex.js

```diff
@@ -79,13 +78,21 @@ const Home = () => {
           }
         });
 
-        genderLists?.femaleList?.pokemon_species_details.forEach((genderItem) => {
-          if (genderItem?.pokemon_species?.name === item.name) {
-            item.gender.push("female");
+        genderLists?.femaleList?.pokemon_species_details.forEach(
```

Comment:
mate I don't think we need this code in this file anymore. We can delete it I think